### PR TITLE
Improve RSS entries for statuses

### PR DESCRIPTION
**Pull request by ClearlyClaire**
- Render polls in both accounts and tags serializers
- Refactor RSS serializers
- Change title preview to include ellipsis when truncated
- Change title preview to show CW instead of toot text
- Add tests

### CODE REVIEW

File: app/models/status.rb [hunk 1]
```diff
@@ -198,10 +198,20 @@ def preview_card
...
 
   def title
-    if destroyed?
-      "#{account.acct} deleted status"
+    return "#{account.acct} deleted status" if destroyed?
+
+    preview = proper.spoiler_text.presence || proper.text
+    if preview.length > 30 || preview[0, 30].include?("\n")
+      preview = preview[0, 30]
+      preview = preview[0, preview.index("\n").presence || 30] + '…'
```
Comment (Gargron) [inline app/models/status.rb:206]: The fact that we're re-implementing ActiveSupport text helpers in here makes me think maybe this method has overstayed its welcome here and maybe it belongs in some presentation layer.

Comment (ThibG) [reply app/models/status.rb:206]: Yeah, I thought about moving it to the RSS serializers at first but forgot it was used by the OEmbed endpoint. I'll think about where to move it (maybe the Formatter)

Comment (Gargron) [reply app/models/status.rb:206]: I'm not sure why the OEmbed endpoint needs titles, to be honest. Twitter's doesn't give tweets titles. We could get rid of it.

Comment (ThibG) [reply app/models/status.rb:206]: Does twitter use OEmbed?

Comment (Gargron) [reply app/models/status.rb:206]: Yeah!

**Review by Gargron — APPROVED** [review]

**Pull request merged**